<?xml version="1.0"?>
<launch> 
  <!-- 
	Valori exposure: IMPOSTARE SU OGNUNA DELLE DUE CAMERE
				* esterno luce solare		   	 
				* esterno con cielo da pioggia 	 
  -->

	 <!-- PHOTONFOCUS VIS CAMERA MODEL MV1-D2048x1088-HS03-96-G2 MAC:00:11:1c:f9:64:6d IP:192.168.4.3 SN:042900054357-->
	 

	 <node pkg="ira_photonfocus_driver" type="ira_photonfocus_driver_mono" name="vis" output="screen">
			<param name="ip" value="192.168.4.3"/>
			<param name="frame_id" value="vis_camera"/>
			<param name="Width" value="2048" />
			<param name="Height" value="1088" />
		  	<param name="ExposureTime" value="10000" />
		  	<param name="AcquisitionFrameRateEnable" value="true" />
		  	<param name="AcquisitionFrameRate" value="15" />
	 </node>

	 
	 
	 <!-- PHOTONFOCUS NIR CAMERA MODEL MV1-D2048x1088-HS02-96-G2-10 MAC:00:11:1c:f9:5b:d4 IP:192.168.3.3 SN:042500054354 -->
	 <!-- 
	 <node pkg="ira_photonfocus_driver" type="ira_photonfocus_driver_mono" name="nir" output="screen">
			<param name="ip" value="192.168.3.3"/>
			<param name="frame_id" value="nir_camera"/>
			<param name="Width" value="2048" />
			<param name="Height" value="1088" />
		  	<param name="ExposureTime" value="70000" />
		  	<param name="AcquisitionFrameRateEnable" value="true" />
		  	<param name="AcquisitionFrameRate" value="15" />
	 </node>
-->

	</launch>
